code/src/components/App.js

```diff
@@ -0,0 +1,71 @@
+/* eslint max-len: ["error", { "code": 180 }] */
+
+import React, { useState } from 'react';
```

Comment:
Just of curiosity, why have you done the App.js as a component and not used the App.js we get in the file?

Reply:
I made a duplicate App.js by mistake and deleted one of them, seems like the one I kept was the new one which somehow had ended up in the components folder.